### PR TITLE
Add details reg. European interoperability to FAQ (#551)

### LINKED ISSUE #551: What does the European interoperability mean? (e.g. the case of Spain)
Hi

I see that now Spain is added to the EU country, where the corona warn app should work. I'd like to clarify this subject. Suppose I am in Spain and I would like to continue using the corona warn app. So

1.  Interchanging keys:. In my understanding the corona warn app and the Spanish radar App would interchange keys, if the conditions are met:

    

-  being near to that person for longer than 15 min

- and being close than 1.5 meter.

2. Downloaded keys of positive cases. Would the corona warn app now when being in Spain, download the keys of positive test cases of the Spanish health system???

3. Positive test case. In case I have a positive test, will the app then connect and upload that information to the spanish health system?

thanks

### PR DESCRIPTION
**Pull request by Ein-Tim**
Closes #551
See @MikeMcC399's comment [here](https://github.com/corona-warn-app/cwa-website/issues/551#issuecomment-724856089)

### CODE REVIEW

**Comment by MikeMcC399** [comment]
Thanks to @Ein-Tim for assisting and apologies for my github-beginner's handling of https://github.com/corona-warn-app/cwa-website/pull/553!

The changes in this PR are:
Add link to EU FAQs https://ec.europa.eu/commission/presscorner/detail/en/qanda_20_1905#gateway and explain where to find participating countries in the app.

Previously discussed with @heinezen in #551.

File: src/data/faq.json [hunk 1]
```diff
@@ -295,7 +295,8 @@
...
                         "active": true,
                         "textblock": [
-                            "Together with the European Commission, an infrastructure for secure information exchange has been developed, known as the \"Interoperability Gateway\". This gateway enables the Corona Warn App to operate across national borders and to receive potential warning messages abroad.",
+                            "Together with the European Commission, an infrastructure for secure information exchange has been developed, known as the \"Interoperability Gateway\". This gateway enables the Corona Warn App to operate across national borders and to receive potential warning messages abroad. Detailed information about the Gateway is available from the European Commission in <a href='https://ec.europa.eu/commission/presscorner/detail/en/qanda_20_1905#gateway'>Coronavirus: EU interoperability gateway for contact tracing and warning apps – Questions and Answers</a>.",
+                            "To view more information in the app, tap EXPOSURE LOGGING ACTIVE (or STOPPED), then tap Transnational Exposure Logging, Participating Countries.",
```
Comment (heinezen) [inline src/data/faq.json:299]: I think it is better to have these instructions in a list format similar to https://www.coronawarn.app/en/faq/#max_power_save

File: src/data/faq_de.json [hunk 2]
```diff
@@ -296,7 +296,8 @@
...
-                            "Gemeinsam mit der EU-Kommission wurde eine Infrastruktur für einen sicheren Informationsaustausch entwickelt, die als sog. \"Interoperability Gateway\" bezeichnet wird. Dieses Gateway ermöglicht, dass die Corona-Warn-App über Landesgrenzen hinaus funktioniert und auch im Ausland potentielle Warnmeldungen empfangen werden können.",
+                            "Gemeinsam mit der EU-Kommission wurde eine Infrastruktur für einen sicheren Informationsaustausch entwickelt, die als sog. \"Interoperability Gateway\" bezeichnet wird. Dieses Gateway ermöglicht, dass die Corona-Warn-App über Landesgrenzen hinaus funktioniert und auch im Ausland potentielle Warnmeldungen empfangen werden können. Detaillierte Informationen zum Gateway bietet die EU-Kommission unter <a href='https://ec.europa.eu/commission/presscorner/detail/de/qanda_20_1905#gateway'>Coronavirus: EU-Datenabgleichsdienst für Kontaktnachverfolgungs- und Warn-Apps – Fragen und Antworten</a>.",
+                            "Um weitere Informationen in der App zu sehen, RISIKO-ERMITTLUNG AKTIV (bzw. GESTOPPT) antippen, dann Länderübergreifende Risiko Ermittlung, Teilnehmende Länder antippen.",
```
Comment (heinezen) [inline src/data/faq_de.json:300]: see above

**Review by heinezen — CHANGES_REQUESTED** [review]
The text looks good to me. I have made some requests about formatting.

Some additional remarks:

* Please give your commits summaries that sum up what you did (not `Updated faq.json`), e.g. `Added link to EU comission's interoperable countries FAQ`. This helps a lot in identifying a specific commit in the git log, especially in case of an error.
* You do not have to click the "Update branch" button every time there is a change. When we merge the PR, we will do that for you :)

**Comment by Ein-Tim** [comment]
@heinezen 

About the requested changes:
Will do this in a few minutes 👍

> Please give your commits summaries that sum up what you did (not Updated faq.json), e.g. Added link to EU comission's interoperable countries FAQ. This helps a lot in identifying a specific commit in the git log, especially in case of an error.

Normally I do, but since this is just the "corrected" version of #553 from @MikeMcC399 I did not to do this time 😅 Sorry

And about the update branch, I apologize, won't do so anymore since you are right about this 👍

File: src/data/faq.json (hunk 1 above)
Comment (Ein-Tim) [reply src/data/faq.json]: Edited it, did you mean like this? Tested the changes via the Chrome Dev Tools (-:

**Comment by heinezen** [comment]
@Ein-Tim If you can (i.e. if you don't use the Github interface), please squash the commits. Then the timeline is a bit cleaner :)

**Comment by Ein-Tim** [comment]
@heinezen I'm using the GitHub Interface, but just out of interest, which Interface can you recommend?

**Comment by heinezen** [comment]
@Ein-Tim I'm probably not allowed to recommend specific software in my role as part of the CWA Open Source Team. You'd have to catch me in private or in one of the other FLOSS projects for that.

However, the git website has a nice list of GUI clients: https://git-scm.com/downloads/guis/

**Comment by svengabr** [comment]
```[13:54:35] 'pages' errored after 1.19 s
[13:54:35] SyntaxError: C:\_dev\workspaces\github\cwa-website\src\data\faq_de.json: Unexpected string in JSON at position 47140
    at parse (<anonymous>)
    at Object.Module._extensions..json (internal/modules/cjs/loader.js:1111:22)
    at Module.load (internal/modules/cjs/loader.js:941:32)
    at Function.Module._load (internal/modules/cjs/loader.js:782:14)
    at Module.require (internal/modules/cjs/loader.js:965:19)
    at require (internal/modules/cjs/helpers.js:88:18)
    at Panini.module.exports [as loadData] (C:\_dev\workspaces\github\cwa-website\node_modules\panini\lib\loadData.js:21:14)
    at Panini.module.exports [as refresh] (C:\_dev\workspaces\github\cwa-website\node_modules\panini\lib\refresh.js:10:8)
    at module.exports (C:\_dev\workspaces\github\cwa-website\node_modules\panini\index.js:41:12)
    at pages (C:\_dev\workspaces\github\cwa-website\gulpfile.js:99:7)```

**Pull request merged**